data/map planets.txt

```diff
@@ -2276,6 +2280,7 @@ planet Ironfirth
 	attributes hai station uninhabited
 	landscape land/station44
 	description `This station was mothballed sometime in the last sixty thousand years. The docking area has an atmosphere, and a status panel indicates that the station's generators are functional but shut down, with standby mechanisms running on solar power alone.`
+	description `	Judging by the size of the docking bay in this station, there was a substantial level of inter-system traffic between Ironfirth and Makerplace. The station's former population most likely communted to the planet for work, filling up its factories and ballooning their output. Considering how busy it must have been in its prime, the docking bay is eerily quiet.`
```

Comment:
```suggestion
	description `	Judging by the size of the docking bay in this station, there appeared to be a substantial level of intra-system traffic between Ironfirth and Makerplace. The station's former population most likely communted to the planet for work, filling up its factories and ballooning their output. Considering how busy it must have been in its prime, the docking bay is eerily quiet.`
```
`Intra-` meaning within the system, as opposed to `inter-` meaning between systems